lib/workflow/stores/mongo.js

```diff
@@ -1059,5 +1064,32 @@ function mongoStoreFactory(waterline, Promise, Constants, Errors, assert, _) {
         return waterline.graphobjects.findOneMongo(query);
     };
 
+    /**
+     * Replace document keywords which start with one string
+     * This is to avoid conflict of keywords between RackHD document and mongodb
+     * @param {Object} data - the document that needs to be transformed
+     * @param {String} beginSrc - the key that starts with this string will be replaced
+     * @param {String} beginDst - the key that will be used in the data base
+     * @memberOf store
+     */
+    exports._replaceKeyword = function(data, beginSrc, beginDst) {
```

Comment:
Actually, IMO, the `beginSrc` and `beginDst` isn't so clear enough. How about below?
```javascript
exports._replaceKeyPrefix = function (data, oldPrefix, newPrefix)
```